trackintel/visualization/locations.py

```diff
@@ -48,27 +60,24 @@ def plot_center_of_locations(locations, out_filename=None, radius=None, position
     locations = transform_gdf_to_wgs84(locations)
 
     if staypoints is not None:
-        staypoints.as_staypoints.plot(radius=staypoints_radius,
-                                      positionfixes=positionfixes,
-                                      plot_osm=plot_osm, axis=ax)
+        staypoints.as_staypoints.plot(radius=staypoints_radius, positionfixes=positionfixes, plot_osm=plot_osm, axis=ax)
     elif positionfixes is not None:
         positionfixes.as_positionfixes.plot(plot_osm=plot_osm, axis=ax)
     elif plot_osm:
-        west = locations['center'].x.min() - 0.03
-        east = locations['center'].x.max() + 0.03
-        north = locations['center'].y.max() + 0.03
-        south = locations['center'].y.min() - 0.03
+        west = locations["center"].x.min() - 0.03
```

Comment:
Is this now formatted with black? 
I doesn't really matter for the visualization files but I think for everything else it is really important that we have a separate pull request for setting everything to black. Otherwise it would be hell to review the pull requests...

Reply:
Yes I agree.. I tried to do that but at the moment we have a lot of tasks ongoing and create that pull request would most likely be a big mess.. Probably best after code stabilizes.
I will close my formatting

Reply:
> we have a lot of tasks ongoing and create that pull request would most likely be a big mess.
--> 💣  ;-)